<?xml version="1.0" encoding="utf-8"?>
<TestRun id="e9e705f4-975d-4ef3-83bb-9d891ea1e066" name="anna@cloudtest-win10 2023-01-22 18:24:41" runUser="cloudtest-win10\anna" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2023-01-22T18:24:41.7864642+00:00" queuing="2023-01-22T18:24:41.7864652+00:00" start="2023-01-22T18:24:33.8210679+00:00" finish="2023-01-22T18:24:41.8268319+00:00" />
  <TestSettings name="default" id="98a142a7-8a64-4126-a4f9-62664464f3ce">
    <Deployment runDeploymentRoot="anna_cloudtest-win10_2023-01-22_18_24_41" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="5dcd9c0f-7d3d-46f1-8394-45d8df26a0e7" testId="07d0ba55-47b1-e83b-d97b-1b52020f0e86" testName="Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_InsertRow" computerName="cloudtest-win10" duration="00:00:02.1137259" startTime="2023-01-22T18:24:41.4719457+00:00" endTime="2023-01-22T18:24:41.4719929+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="5dcd9c0f-7d3d-46f1-8394-45d8df26a0e7">
      <Output>
        <ErrorInfo>
          <Message>InsertRow failed&#xD;
Expected: True&#xD;
Actual:   False</Message>
          <StackTrace>   at UnitTests.MembershipTests.MembershipTableTestsBase.MembershipTable_InsertRow(Boolean extendedProtocol) in C:\Users\anna\orleans\test\TesterInternal\MembershipTests\MembershipTableTestsBase.cs:line 155&#xD;
   at Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_InsertRow() in C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\AzureMembershipTableTests.cs:line 76&#xD;
--- End of stack trace from previous location ---</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_InsertRow" storage="c:\users\anna\orleans\test\extensions\testerazureutils\bin\debug\net7.0\tester.azureutils.dll" id="07d0ba55-47b1-e83b-d97b-1b52020f0e86">
      <Execution id="5dcd9c0f-7d3d-46f1-8394-45d8df26a0e7" />
      <TestMethod codeBase="C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\bin\Debug\net7.0\Tester.AzureUtils.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="Tester.AzureUtils.AzureMembershipTableTests" name="MembershipTable_Azure_InsertRow" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="07d0ba55-47b1-e83b-d97b-1b52020f0e86" executionId="5dcd9c0f-7d3d-46f1-8394-45d8df26a0e7" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" passed="0" failed="1" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.5+1caef2f33e (64-bit .NET 7.0.2)&#xD;
[xUnit.net 00:00:01.27]   Discovering: Tester.AzureUtils (method display = ClassAndMethod, method display options = None)&#xD;
[xUnit.net 00:00:01.48]   Discovered:  Tester.AzureUtils (found 246 test cases)&#xD;
[xUnit.net 00:00:01.50]   Starting:    Tester.AzureUtils (parallel test collections = off, max threads = 4)&#xD;
[xUnit.net 00:00:04.65]       InsertRow failed&#xD;
[xUnit.net 00:00:04.66]       Expected: True&#xD;
[xUnit.net 00:00:04.66]       Actual:   False&#xD;
[xUnit.net 00:00:04.66]       Stack Trace:&#xD;
[xUnit.net 00:00:04.66]         C:\Users\anna\orleans\test\TesterInternal\MembershipTests\MembershipTableTestsBase.cs(155,0): at UnitTests.MembershipTests.MembershipTableTestsBase.MembershipTable_InsertRow(Boolean extendedProtocol)&#xD;
[xUnit.net 00:00:04.66]         C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\AzureMembershipTableTests.cs(76,0): at Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_InsertRow()&#xD;
[xUnit.net 00:00:04.66]         --- End of stack trace from previous location ---&#xD;
[xUnit.net 00:00:04.69]   Finished:    Tester.AzureUtils&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="cloudtest-win10" outcome="Min" timestamp="2023-01-22T18:24:41.4668201+00:00">
        <Text>[xUnit.net 00:00:04.65]     Tester.AzureUtils.AzureMembershipTableTests.MembershipTable_Azure_InsertRow [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="cloudtest-win10" outcome="Warning" timestamp="2023-01-22T18:24:41.6342379+00:00">
        <Text>Data collector 'Blame' message: All tests finished running, Sequence file will not be generated.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>